cwms-data-api/src/test/java/cwms/cda/api/LocationGroupControllerTestIT.java

```diff
@@ -36,32 +44,69 @@
 import cwms.cda.data.dto.LocationGroup;
 import cwms.cda.formatters.ContentType;
 import cwms.cda.formatters.Formats;
+import usace.cwms.db.jooq.codegen.packages.CWMS_ENV_PACKAGE;
 
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.Collections;
 import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 
 import static cwms.cda.api.Controllers.*;
 import static io.restassured.RestAssured.given;
 import static org.hamcrest.Matchers.*;
 
 @Tag("integration")
-@Disabled()
 class LocationGroupControllerTestIT extends DataApiTestIT {
+    private static final Logger LOGGER = Logger.getLogger(LocationGroupControllerTestIT.class.getName());
+    private List<LocationGroup> groups = new ArrayList<>();
```

Comment:
It looks like these field as specifically used for cleanup operations. Rename the field variable `groupsToCleanup` or something similar for clarity.

Reply:
Renamed as suggested.